extension.js

```diff
@@ -241,7 +241,7 @@ const TrackerSearchProvider = new Lang.Class({
     },
 
     filterResults : function(results, max) {
```

Comment:
Well, that's my question in the pull request. I don't know exactly what means this max and if we should take it into account (I've made the assumption it was a function signature needed by the Tracker Search Provider API, but I didn't found it).

I've made this pull request, because I'm sure about at least one thing: '5' is a magic number and it isn't defined by the user (on the contrary of MAX_RESULTS).